src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,11 +1,27 @@
-export const TodoFilter = () => (
+import React from 'react';
+import { TaskStatus } from '../../types/types';
+
+interface Props {
+  query: string;
+  setQuery: (_: string) => void;
+  setTaskStatusFilter: (_: TaskStatus) => void;
```

Comment:
`_` is a bad name, you can use it only if you need to skip the argument and use the next one, in this case it doesn't make sense